docs/customresourcestate-metrics.md

```diff
@@ -253,11 +253,6 @@ kube_customresource_ref_info{customresource_group="myteam.io", customresource_ki
 In v2.9.0 the `vericalpodautoscalers` resource was removed from the list of default resources. In order to generate metrics for `verticalpodautoscalers`, you can use the following Custom Resource State config:
```

Comment:
Line 253 should be `verticalpodautoscalers` instead of `vericalpodautoscalers`

Reply:
resolved